biome.json

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+  "vcs": {
+    "clientKind": "git",
+    "enabled": true,
+    "useIgnoreFile": true
+  },
+  "files": {
+    "ignoreUnknown": true
+  },
+  "formatter": {
+    "enabled": true,
+    "formatWithErrors": false,
+    "indentStyle": "space",
+    "indentWidth": 2,
+    "lineEnding": "lf",
+    "lineWidth": 80,
+    "ignore": [
+      "**/package.json",
+      "**/dist/**",
+      "**/.changeset",
+      "**/CHANGELOG.md",
+      "**/pnpm-lock.yaml",
+      "**/pnpm-workspace.yaml"
+    ]
+  },
+  "organizeImports": { "enabled": true },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "recommended": true,
```

Comment:
I migrated the eslint file which added a lot of rules so I compared these rules to the list of recommended rules by biome and then removed all the redundant and decided to enable all the recommended rules.

This ensures that we follow the recommended rules and have a clean config file for biome.

find the recommended rules here https://biomejs.dev/linter/rules/#recommended-rules